### ISSUE TITLE
PACKAGE-MANGER naming inconsistent

### ISSUE THREAD

**Issue by kestewart** [spdx/spdx-spec#58]
**Yev Bronshteyn 2016-05-30:**
The  "PACKAGE-MANAGER" category is inconsistent with other names, where we use underscore instead of hyphen (such as "DISTRIBUTION_ARTIFACT" or "DATAFILE_OF" in relationship).

The categories are not demonstrated in the RDF examples. To demonstrate them, we would need to, ideally, represent them with URIs, e.g.

```
<category rdf:resource="http://spdx.org/rdf/terms#referenceCategory_package_manager" />
```
			
This also means categories need to be added to the ontology. 

Lastly, upon further reading, I would recommend separating the "target" property in RDF into two: "type" and "locator", which are terms we already define spearately. Unliked the tag format, which aims to be readable, the core tenet of RDF is to be resolvable. This way, type can be represented in RDF by a URI that can resolve to provide more information about the target. We can define the vocabulary of that as part of the ontology work for SPDX 2.1 - it needn't be in the spec.

So an example of a full external reference in to a standard repository might be:

```
<spdx:Package  rdf:about="http://yevster.com/packages/foobar">
	<spdx:externalRef>
		<spdx:ExternalRef>
                    <spdx:referenceCategory rdf:resource="http://spdx.org/rdf/terms#referenceCategory_package_manager" />
		    <spdx:referenceType rdf:resource="http://spdx.org/rdf/refeferences/maven-central" />
                    <spdx:referenceLocator>org.apache.commons:commons-lang:3.2.1</spdx:referenceLocator>
		</spdx:ExternalRef>
	</spdx:externalRef>
</spdx:package>
```

**Yev Bronshteyn 2016-05-31 04:55:57 UTC**
It should be pointed out that the approach described for external reference types above is the same one that we use for all other "listed values", including license IDs, relationship types, etc. Anything that's listed in the spec (the body or appendix) is identified by a URI in RDF format. I submit that this should also be the case for reference types.

Note:  This was https://bugs.linuxfoundation.org/show_bug.cgi?id=1361

**Comment by wking** [comment]
With the approach #54 takes to addressing #53, the old `type` and `locator` are being collapsed into a single `uri`, which would make at least some of this issue obsolete.  I'll update #54 to more explicitly define the category resource URIs, and after that I think #54 will completely cover this issue.

**Comment by kestewart** [comment]
This is a bug,  and inconsistent between 2 places (3.21& Appendix VI) in the spec.    Needs pull request.

**Comment by kestewart** [comment]
This was actually fixed in 2.1.1 - see https://github.com/spdx/spdx-spec/issues/27
